bin/phyx2owl.js

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const phyx = require('..');
+
+/*
+ * An application for converting input Phyx files to OWL ontologies.
+ */
+
+// Read command line arguments.
+const argv = require('yargs')
+  .usage("$0 [files or directories to convert into OWL ontologies]")
+  .describe('max-internal-specifiers', 'The maximum number of internal specifiers (phylorefs with more than this number will be ignored)')
+  .default('max-internal-specifiers', 8)
+  .describe('max-external-specifiers', 'The maximum number of external specifiers (phylorefs with more than this number will be ignored)')
+  .default('max-external-specifiers', 8)
+  .help()
+  .alias('h', 'help')
+  .argv
+
+/*
+ * Get a list of all files in a directory. We will recurse into directories and choose
+ * files that meet the criteria in the function `check(filename) => boolean`.
+ */
+function getFilesInDir(filePath, check = (filename => filename.toLowerCase().endsWith(".json"))) {
+  // console.debug(`Processing file: ${dir}`)
+  if (!fs.existsSync(filePath)) return [];
```

Comment:
So currently giving a non-existing directory (or file) would result in nothing converted but otherwise no error. Shouldn't this be an error condition reported to the user as feedback?

Reply:
Good catch! I fixed this in 4c966394a3. I also realized that I didn't catch this earlier because I hadn't written tests for phyx2owl.js, so I added that in 4d3e1a5b5d53 as well.